scripts/basic-workflow-tests.sh

```diff
@@ -133,13 +133,24 @@ test_generate() {
 test_train() {
     task Train the model
 
+    # Reduce testfile to 1 sample, generating from the new model is extreemly slow in CI
+    # generating only a single test should be enough as we're not verifying the output
+    TESTFILE=$(set +f ; ls -tr generated/test_*jsonl | tail -n 1)
+    mv $TESTFILE ${TESTFILE}_
+    head -n 1 ${TESTFILE}_ > $TESTFILE
+    rm ${TESTFILE}_
```

Comment:
README suggests these files can have spaces in the names? Probably should quote `$TESTFILE`

you could also do this without a explicit temp file if you used sed, e.g.,

```
sed -ie '2,$d' "${TESTFILE}"
```